tests/unit/amo/components/TestAddonCompatibilityError.js

```diff
@@ -10,25 +10,30 @@ import {
   INCOMPATIBLE_NOT_FIREFOX,
   INCOMPATIBLE_OVER_MAX_VERSION,
   INCOMPATIBLE_UNDER_MIN_VERSION,
+  INCOMPATIBLE_UNSUPPORTED_PLATFORM,
 } from 'core/constants';
-import { signedInApiState } from 'tests/unit/amo/helpers';
-import { fakeI18n, shallowUntilTarget } from 'tests/unit/helpers';
+import { dispatchClientMetadata } from 'tests/unit/amo/helpers';
+import {
+  fakeI18n, shallowUntilTarget, userAgentsByPlatform,
+} from 'tests/unit/helpers';
 
 
 describe(__filename, () => {
-  const defaultUserAgentInfo = {
-    browser: { name: 'Firefox' },
-    os: { name: 'Plan 9' },
+  let store;
+
+  beforeEach(() => {
+    store = createStore().store;
```

Comment:
This could use `dispatchClientMetadata` for a more accurate/complete "default" store.

Reply:
This component relies on `lang` and `userAgent` so I wanted each test to call `dispatchClientMetadata()` instead since the values are important. I can add a comment to this if it's helpful though.